docker-compose.yml

```diff
@@ -1,50 +1,76 @@
-version: '2.1'
+version: '3'
+
+# These are the PRODUCTION defaults for docker-compose.yml
+# It is intended to be run in production mode via Docker stack, not docker-compose.
+# There are development overrides specified in docker-compose.override.yml, which are intended to be run in docker-compose.
+# In addition, your system should have three files for environmental variables, none of which should be committed to git:
+#  .env
+#  .env.produciton
+#  .env.development
 
 services:
   db:
+    image: cottagelabs/willow-db
     build: db/
-    env_file: .env
+    env_file:
+      - .env
     volumes:
       - postgres_data:/var/lib/postgresql/data
-  
+      - ./db/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d:ro
```

Comment:
Volume full of static scripts in production? It might be best to copy them into the image. I think we probably will not want to supply those scripts from the host server running docker. We might not even be able to touch the host in some production deployments of Willow.

Of course this is fine for development.